sdk/src/gateway/client.ts

```diff
@@ -1007,6 +1007,9 @@ export class GatewayApiClient {
         );
         const orders: OnrampOrderResponse[] = await response.json();
         return orders.map((order) => {
+            const outputTokenAddress = order.outputTokenAddress ?? order.tokensReceived?.[0]?.tokenAddress ?? undefined;
+            const outputTokenAmount = order.outputTokenAmount ?? order.tokensReceived?.[0]?.amount ?? undefined;
```

Comment:
`order.tokensReceived?.[0]`
I think it should be the last token in the list and not the first one, right?

Reply:
tokensReceived only contains tokens received by the recipient. most of the time it should contains one token, it could contains more if the user receive multiple tokens